.scala-steward.conf

```diff
@@ -0,0 +1,6 @@
+postUpdateHooks = [{
+  command = ["sbt", "githubWorkflowGenerate"],
+  commitMessage = "Regenerate workflow with sbt-github-actions",
+  groupId = "ch.epfl.scala",
+  artifactId = "sbt-scalafix"
+}]
```

Comment:
Since scalafix sets the Scala 2.13 version, we may need to regenerate workflows when scalafix updates because that may also update Scala 2.13 for us.

Reply:
Nice